Integrations/src/main/java/io/deephaven/integrations/python/PythonFunction.java

```diff
@@ -65,6 +68,29 @@ public PythonFunction(final PyObject pyObjectIn, final Class classOut){
         }
     }
 
+
+    // these methods call Python functions and return them as PyObjects without attempting to wrap them in java types
+    // I'm sure there's a cleaner way to do this?
+    public PyObject passThrough(T[] t) {
+        PyObject out = pyCallable.call("__call__", t);
+        return out;
+    }
+
+    public PyObject passThrough(T t) {
+        PyObject out = pyCallable.call("__call__", t);
+        return out;
+    }
+
+    public PyObject passThroughGather(IndexSet idx, ColumnSource<?>[] colSet) {
+        PyObject out = pyCallable.call("__call__", idx, colSet);
+        return out;
+    }
+
+    public PyObject passThroughScatter(PyObject result, long k) {
+        PyObject out = pyCallable.call("__call__", result, k);
+        return out;
+    }
```

Comment:
This has a bad code smell.  

1) `PythonFunction` is a general purpose class.  This stuff that was added is very specific purpose.  It doesn't belong here.
2) Is there a reason not to just use the apply method?
3) If these helper methods need to exist, they should be in some AI specific place, because that's all they are for.